app/src/main/java/org/mozilla/focus/FocusApplication.kt

```diff
@@ -47,6 +48,7 @@ open class FocusApplication : LocaleAwareApplication(), CoroutineScope {
 
             TelemetryWrapper.init(this)
```

Comment:
Wouldn't it be nice to only initialize all these only if `isTelemetryEnabled()` and then observe changes and if the users enable telemetry do the initialization or immediately stop the functionality?
Currently the first two will do their own `isTelemetryEnabled()` check and abort the initialization but the `FactsProcessor` won't.